workflow:
  auto_cancel:
    on_job_failure: unexpected_value
